### PR TITLE
Another Flatpak/KDE5 wallet workaround

### PR DESCRIPTION
**Pull request by Sarke**
Another Flatpak/KDE5 wallet workaround, it worked well for me.  I think this is the more correct way to do it.

### CODE REVIEW

**Comment by Sarke** [comment]
Any reason for just closing it?

**Comment by Sarke** [comment]
I had the same problem, and searching found this: https://github.com/microsoft/vscode-docs/pull/6577

And this: https://github.com/microsoft/vscode/issues/189672

I ended up using the permission settings for Flatpak instead of a workaround through the Gnome library, so I thought I'd share.

**Comment by ntrogh** [comment]
> Any reason for just closing it?
@Sarke apologies, I inadvertently closed the PR! Back to reviewing it and pressing the right buttons.

Appreciate your contribution!

**Comment by ntrogh** [comment]
@TylerLeonhardt Can you have a quick look at this docs update? LGTM.

File: docs/editor/settings-sync.md [hunk 1]
```diff
@@ -195,6 +195,11 @@ It's possible that your wallet (aka keyring) is closed. If you open [KWalletMana
 
 If you are using KDE5 or higher and are having trouble connecting to `kwallet5` (like users of the unofficial VS Code Flatpak in [issue #189672](https://github.com/microsoft/vscode/issues/189672)), you can try [configuring the keyring](#other-linux-desktop-environments) to `gnome-libsecret` as this will use the [Secret Service API](https://www.gnu.org/software/emacs/manual/html_node/auth/Secret-Service-API.html) to communicate with any valid keyring. `kwallet5` implements the Secret Service API and can be accessed using this method.
 
+Another Flatpak/KDE5 fix you can try is granting the specific D-Bus service persmissions:
```
Comment (ntrogh) [inline docs/editor/settings-sync.md:198]:
```suggestion
If you're still experiencing trouble connecting to `kwallet5`, some users have reported that granting the specific D-Bus service permissions proved a viable fix:
```

**Review by ntrogh — CHANGES_REQUESTED** [review]
@Sarke Suggest to alter the language to make it clear that this fix is not verified by the team. Thanks a lot for contributing and helping others be successful!

File: docs/editor/settings-sync.md (hunk 1 above)
Comment (Sarke) [reply docs/editor/settings-sync.md:198]: Sure, but maybe we can combine it with the suggested workaround in the paragraph above, which is also Flatpak and KDE related, and I think also not verified by the team?

Comment (Sarke) [reply docs/editor/settings-sync.md:198]: In a new section.

Comment (Sarke) [reply docs/editor/settings-sync.md:198]: Oh, that PR was a MS member.

**Review by ntrogh — APPROVED** [review]
LGTM! Thanks a lot @Sarke

**Pull request merged**